Makefile

```diff
@@ -116,11 +116,15 @@ help:  ## Display this help
 
 .PHONY: test
 test: ## Run tests.
+	source ./scripts/fetch_ext_bins.sh; fetch_tools; setup_envs; go test ./... $(TEST_ARGS)
+
+.PHONY: test-verbose
+test-verbose: ## Run tests with verbose settings.
 	source ./scripts/fetch_ext_bins.sh; fetch_tools; setup_envs; go test -v ./... $(TEST_ARGS)
 
 .PHONY: test-cover
-test-cover: ## Run tests with code coverage and code generate  reports
-	source ./scripts/fetch_ext_bins.sh; fetch_tools; setup_envs; go test -v -coverprofile=out/coverage.out ./... $(TEST_ARGS)
+test-cover: ## Run tests with code coverage and code generate reports.
```

Comment:
The comments are all inconsistent in this file. Some are sentences, some have `.` some have neither. I didn't make it consistent here because I didn't want to expand this PR too much.

Reply:
This would be a nice follow up/good-first issue...